tests/util/block.ts

```diff
@@ -0,0 +1,28 @@
+import { ApiPromise } from "@polkadot/api";
+import { BlockHash } from "@polkadot/types/interfaces/chain";
+
+export async function createAndFinalizeBlock(
```

Comment:
Some comments on this util function would be nice. The name is pretty self documenting, nut explaining that it's returning the creation time and that an optional parent hash and finalize param exist would be nice.